README.md

```diff
@@ -2,22 +2,32 @@
 [![CI](https://github.com/ansible-collections/community.sops/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/community.sops/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.sops)](https://codecov.io/gh/ansible-collections/community.sops)
 
 <!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+The `community.sops` collection allows integrating [`mozilla/sops`](https://github.com/mozilla/sops) in Ansible.
 
+`mozilla/sops` is a tool for encryption and decryption of files using secure keys (GPG, KMS). It can be leveraged in Ansible to provide an easy to use and flexible to manage way to manage ecrypted secrets' files.
+ 
 ## Tested with Ansible
 
 <!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
 
+- `devel`
+
```

Comment:
Can we please add 2.10 here and into CI testing?

Reply:
Should be simpe to do by adding a matrix variable which has values `devel` and `stable-2.10` (and maybe also `stable-2.9`). It should be used in lines https://github.com/ansible-collections/community.sops/blob/main/.github/workflows/ansible-test.yml#L21 and https://github.com/ansible-collections/community.sops/blob/main/.github/workflows/ansible-test.yml#L74

Reply:
Can copy from https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml